Feature: Adds ID property (#96)
flekschas/svelte-simple-modal: README.md

</h1>

<div align="center">
  
  **A simple, small, and content-agnostic modal for [Svelte](https://svelte.dev).**

>   

Reviewer: Please undo these and the following unnecessary removal of indentations.

Author: My bad, guess my autolinter caught em. Will fix!

The `<Modal />` component accepts the following optional properties:

| Property                  | Type                                                 | Default             | Description                                                                                                                                                                                                                                                 |
| ------------------------- | ---------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **show**                  | Component \| null                                    | `null`              | A Svelte component to show as the modal. See [Store API](#store-api) for details.                                                                                                                                                                           |

> | **id**                    | string \| null                                       | `null`              | Element ID to be applied to the modal. Useful for interfacing with the modal from JavaScript or CSS.                                                                                                                                                        |

Reviewer: ```suggestion
| **id**                    | string \| null                                       | `null`              | Element ID to be assigned to the modal's root DOM element.                                                                                                                                                        |
```
